src/reference-manual/expressions.Rmd

```diff
@@ -156,7 +156,9 @@ or `add`, but it may be named `pi` or `e`.
 Variable names will also conflict with the names of distributions
 suffixed with `_lpdf`, `_lpmf`, `_lcdf`, and `_lccdf`, `_cdf`, and
 `_ccdf`, such as `normal_lcdf_log`; this also holds for the deprecated
-forms `_log`, `_cdf_log`, and `_ccdf_log`.
+forms `_log`, `_cdf_log`, and `_ccdf_log`. No user-defined variable
```

Comment:
Hm, we need to revise this section actually. As of 2.25 Stan Math function names are allowed as identifiers. The only restriction is that identifier can not end with _lupdf/_lupmf 

So allowed: add, normal_lpdf, sum,...
Not allowed: noraml_lupdf, p_lupmf

Reply:
Feel free to push directly into this

Reply:
Will do!